<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCalendarWidget" name="calendar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_events">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Events</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>522</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Events on Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="event_list"/>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Add Event</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="2">
             <widget class="QPushButton" name="del_button">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QLineEdit" name="event_title"/>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="add_button">
              <property name="text">
               <string>Add/Update</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QTextEdit" name="event_detail"/>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="event_category">
              <item>
               <property name="text">
                <string>New...</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Work</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Doctor</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Meeting</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTimeEdit" name="event_time">
              <property name="time">
               <time>
                <hour>12</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_tasks">
       <attribute name="title">
        <string>Tasks</string>
       </attribute>
       <widget class="QLineEdit" name="taskLineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnaddNewTask">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add new</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnsaveChanges">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>361</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save changes</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>41</y>
          <width>361</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
